census_data_aggregator/__init__.py

```diff
@@ -405,3 +406,104 @@ def approximate_percentchange(pair_old, pair_new):
 
     # Return the results
     return percent_change_estimate, percent_change_moe
+
+
+def approximate_mean(range_list, number_replicates=50, usePareto=False):
+    """
+    Estimate a mean and approximate the margin of error. The Census Bureau guidelines do not provide instructions for
+    approximating a mean using data from the ACS. They do provide guidance for approximating a mean with data `from the PUMS`_.
+    Instead, we implement a simulation based approach. Due to the stochastic nature of the simulation approach, you will need to set
+    a seed before running this function to ensure replicability. Note that this function assumes you have a lower bound for the smallest
+    bin and an upper bound for the largest bin. We recommend trying different lower and upper bounds to assess the sensitivity of the
+    resulting mean to your assumptions.
+
+    Args:
+        range_list (list): A list of dictionaries that divide the full range of data values into continuous categories.
+            Each dictionary should have four keys:
+                * min (int): The minimum value of the range
+                * max (int): The maximum value of the range
+                * n (int): The number of people, households or other unit in the range
+                * moe (float): The margin of error for n
+        number_replicates (int): number of replicates for simulation, used to estimate margin of error
```

Comment:
Let's call this `simulations` and avoid the use of "replicates. We should also list the default here.